scripts/clean

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env bash
 
 for f in packages/*; do
-  if [[ -f ${f}/package.yaml ]]; then
-  	if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
-  		rm -rf ${f}/charts
-		rm -rf ${f}/charts-crd
-  	fi
-  fi
+	if [[ -f ${f}/package.yaml ]]; then
+		if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
+			if [[ $(cat ${f}/package.yaml | yq r - url) ]]; then
+				rm -rf ${f}/charts
+				rm -rf ${f}/charts-crd
+			else
+				[ -e ${f}/charts-crd ] && mv ${f}/charts-crd/templates ${f}/charts/crds
```

Comment:
This logic will need to be modified when https://github.com/rancher/charts/pull/595 gets merged in since the CRDs that need to be moved back will be in the `crd-manifest/` directory instead of the `template/` directory.

Since the logic for reverting a CRD chart might need to be exposed here, perhaps it might make sense to take the function currently located in generated-patch and make it its own script (i.e. `clean-crds`) and call that here instead:
https://github.com/rancher/charts/blob/4e030b0ca7ae349ea3b28c4593b00d142da2b80e/scripts/generate-patch#L18-L33

Would also make sense along the way to add in this check directly in that script:
https://github.com/rancher/charts/blob/4e030b0ca7ae349ea3b28c4593b00d142da2b80e/scripts/generate-patch#L81-L84

Reply:
I thought that crd-manifest thing was optional ? We need to talk about that in other PR, I don't think all of them should follow that route. So lets just merge this PR before that one.

Reply:
The `crd-manifest` stuff is optional, not all charts will follow this route but either this PR or https://github.com/rancher/charts/pull/595 will need to contain this change. If we plan to merge this PR in before, I can add the necessary changes to the other PR instead